easybuild/easyblocks/c/cp2k.py

```diff
@@ -803,6 +806,20 @@ def install_step(self):
         except OSError, err:
             raise EasyBuildError("Copying executables from %s to bin dir %s failed: %s", exedir, targetdir, err)
 
+        # copy libraries
+        if self.cfg['library']:
+            targetdir = os.path.join(self.installdir, 'lib')
+            libdir = os.path.join(self.cfg['start_dir'], 'lib/%s' % self.typearch, self.cfg['type'])
+            try:
+                if not os.path.exists(targetdir):
+                    os.makedirs(targetdir)
+                os.chdir(libdir)
+                for library in os.listdir(libdir):
+                    if os.path.isfile(library):
+                        shutil.copy2(library, targetdir)
```

Comment:
@ocaisa Please use `change_dir` and `copy_file` from `filetools` module here.
Also no need to pre-create the `targetdir`, `copy_file` already does that.

```python
try:
    change_dir(libdir)
    for lib in os.listdir(libdir):
        if os.path.isfile(lib):
            copy_file(lib, os.path.join(targetdir, lib))

except OSError as err:
    raise EasyBuildError("...")